src/inventory_ui.h

```diff
@@ -254,67 +325,92 @@ class inventory_selector
         /** Returns an entry from @ref entries by its invlet */
         inventory_entry *find_entry_by_invlet( long invlet ) const;
 
+        const std::vector<inventory_column *> &get_all_columns() const {
+            return columns;
+        }
+        std::vector<inventory_column *> get_visible_columns() const;
+
         inventory_column &get_column( size_t index ) const;
         inventory_column &get_active_column() const {
             return get_column( active_column_index );
         }
 
         void set_active_column( size_t index );
-        size_t get_columns_width() const;
+        size_t get_columns_width( const std::vector<inventory_column *> &columns ) const;
+        /** @return percentage of the window occupied by columns */
+        double get_columns_occupancy_ratio() const;
+        /** @return do the visible columns need to be center-aligned */
+        bool are_columns_centered() const;
+        /** @return true if visible columns are wider than available width */
```

Comment:
Might yield ugly doxygen. Rephrase similar to *Are visible columns wider than available width?*